### DISCUSSION TITLE
Scrollable nested menu's

### DISCUSSION THREAD

**Discussion by ahmedy00** [vuetifyjs/vuetify#20473] [Q&A]
Good day everyone,

I needed some nested menu in my app so I created. You may see the [reproduction](https://play.vuetifyjs.com/#eNqVVE2P2jAQ/SujbCVYCQPa3hCgfpx66KXqbVkJr+OAu45t2RMKQvz3jh3IEpLtai9RPPP85ut5Ho9Z8GLy1bnxrpLZLJujLJ3mKJcrAzDfMe5c+k0HYQ1yZaQHoXkIi1WWs0LLPfypAqriwKTJV9kZn26U0lSgreCorCG8lgWuMpgJbYNk1iRKaZAJrcQLAQqug7ziIJZLSnDHBaodR+sJeHTeunBqQVPIZzSgiBboV5mcoAl5A0zQCFuWuWK5xcB20qMSXM8nZ0+LmIzEfJ3XpNWrhlRwnzf9cXs2BcfZQzdPrQJCyfdsK9VmiwT+PJ26fV+eEcoUhaOailR9PIyAyqPmKwMP09jUF3mIrmjssPx3GhyRiy0Z1ndUI/t0TByndQ/LB+fRnkufBzpz6kd9uShE2823CtGaX7J4C+xlQcjn92AzFcOub2ruxy6bYs9S6IP1tatPJh25WJ2auPFSGmoh/DDxlX2PvhgugroDJUec6K1eyNwIpqPh6GmJ+Ib8hjT5L6++3glkqZdCqzI6BuGVQwgSq3pnqNJZj3CM4xiBNT9tRY89H4GRe/xN04QTFN6WMKDlM4g3KBQ9imZusIhXh49P9zEAvFIMeTgYAcN7WCzhWKfK/3KFDfWQrkRrZLRajkk1wwEthbImGIxeo4x3XFcy4U/nQHUe10qjVFrRWrwN1Vu0sVl1e6gx2WlkKq3T5+kfBjqfeA==). Also see the code below:

```
  <v-app>
    <v-container class="d-flex justify-end">
      <v-menu location="left" :close-on-content-click="false">
        <template #activator="{props}">
          <v-btn icon v-bind="props">
            <v-icon>mdi-dots-vertical</v-icon>
          </v-btn>
        </template>
        <v-card class="px-0 pa-2">
          <v-list max-height="300px">
            <v-list-item v-for="item, index in 20" :key="index">
              <v-menu location="left" :attach="`#btn-${index}`">
                <template #activator="{props}">
                  <v-btn
                    v-bind="props"
                    @click="logButtonRef"
                    ref="buttonRef"
                    :id="`btn-${index}`"
                    >activator</v-btn
                  >
                </template>
                <v-card color="green"> Inner Card </v-card>
              </v-menu>
            </v-list-item>
          </v-list>
        </v-card>
      </v-menu>
    </v-container>
  </v-app>
</template>

<script setup>
  import { ref, onMounted, nextTick } from 'vue'
  const buttonRef = ref([])

  onMounted(async () => {
    await nextTick()
    console.log('on mounted', buttonRef.value)
  })

  const logButtonRef = () => {
    console.log('buttonRef', buttonRef.value)
  }
</script>
```
After creating the nested menus, I attached the innermost menu to its own button for the scrollable state and made it stay where it was first created when scrolled, but this caused a problem like this:

I can no longer place the inner menu to the left of its own button. I tried v-menu's offset attribute, but I couldn't get it to work. Also, when I wrote custom style, it didn't produce any results.
I would greatly appreciate any suggestions you may have.

Tried set v-menu's location attribute to left and doesn't work.

Tried set v-menu's offset attribute to a number array(like [-20, -20] etc.)

Tried give some static margin and transform: translateX()

**Comment by angrymarker** [comment]
Removing :attach="`#btn-${index}`" worked for me
https://play.vuetifyjs.com/#eNqVVE1v2zAM/SuENyApECVBdwuSYB+nHXYZdmsKVJXpVKssCZKcJQj830fJiRvHzoZeDIt8eiQfST0cM+/E7Iu1012F2SJbBiyt4gHXGw2w3DFubfpNB2F04FKjA6G496tNlrNC4R5+Vz7I4sBQ55vshE83StQVKCN4kEYTXmERNhkshDIemdGJEnVgQknxSoCCK48XHMRyTgk+cBHkjgfjCHi0zlhfd6Ap5HPQIIkW6FfqnKAJeQVM0Ahbl7lkuQme7dAFKbhazk6eDjEZifkyr1lHq5ZUcJe3+tg9m4Pl7L6fp5I+QMn37AXl9iUQ+NN8bvdDeUYokxSOaipS9fEwASqPxJca7udR1Fc8RFc09lj+0Y0e8p2ad7Uf8kCvF8Ooz+cpUGb7tQrB6J9Y3AI7LAj5/D/YQsawT5QZ+3hM2tRPt7DrtthTu4dgQ3INjUJvJIxKIm4doiYJ4buOm/Qt+mK4COo3jRyxa9czQeZ2KHpzGj2dQb0ivyJN/vNmN3tPlmbxO5XR0QsnbQCPoWreBVla4wIcYzsmYPQPU9FC5xPQuA+/qJtQQ+FMCSN6YEbxBoWiwW/7Bqt4dfzweBcDwBvFmPuDFjC+g9Uajk2q/A+XoaUe05VojYxG4ZSmZjyixS8bgtHkLcp0x1WFCV+fAjV5XE4apdKJ1uFtqW7RRrEaeUiYrJ7oSqn0efwL8FWV6A==

**Reply by ahmedy00** [reply]
Hey @angrymarker You're right, but I need it attached so that it stays when it was first rendered position when I scroll the elements. But thank you anyway
